rust/otap-dataflow/crates/otap/src/otlp_exporter.rs

```diff
@@ -40,14 +53,16 @@ pub const OTLP_EXPORTER_URN: &str = "urn:otel:otlp:exporter";
 #[derive(Debug, Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct Config {
-    /// The gRPC endpoint to connect to
-    pub grpc_endpoint: String,
-    /// The compression method to use for the gRPC connection
-    pub compression_method: Option<CompressionMethod>,
-    /// Timeout for RPC requests. If not specified, no timeout is applied.
-    /// Format: humantime format (e.g., "30s", "5m", "1h", "500ms")
-    #[serde(default, with = "humantime_serde")]
-    pub timeout: Option<Duration>,
+    /// Shared gRPC client settings reused across OTLP exports.
+    #[serde(flatten)]
+    pub grpc: GrpcClientSettings,
+    /// Maximum number of concurrent in-flight export RPCs.
+    #[serde(default = "default_max_in_flight")]
+    pub max_in_flight: usize,
```

Comment:
Maybe we should the name of the field or the documentation reflect that this is a per-core setting?